src/utils.jl

```diff
@@ -0,0 +1,10 @@
+import Base
+
+function sum_if_not_none_or_initialize(A,B)
+    if isnothing(A) 
+        return deepcopy((B))
+    end
+    return A + B
+end
+
+sqrt(x::Real) = x < 0.0 ? NaN : Base.sqrt(x)
```

Comment:
I previously wrote a comment (in the wrong page, so I deleted it) that this was overriding a `Base` method.  This wasn't true, you're instead shadowing it (less bad, but still a bit).  What's the use case for redefining `sqrt` in this way?  If the argument is negative, an error is usually the right thing to do, instead of poising everything with `NaN`, which is hard to debug.

Reply:
Well I asked Matteo on it, [Slack Message](https://stingraysoftware.slack.com/archives/C03GEPZ9ZB7/p1657274550265329?thread_ts=1657272098.400529&cid=C03GEPZ9ZB7)